app/routes/_layout.tsx

```
@@ -0,0 +1,30 @@
+import { Header } from '~/components/layout/header';
+import { Outlet } from 'react-router';
+import type { Route } from '../../.react-router/types/app/routes/+types/home';
+import { ThemeProvider, useTheme } from '~/components/theme';
+
+export function meta({}: Route.MetaArgs) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)